woocommerce/woocommerce-gateway-stripe
# Remove Link beta headers

Fixes: #2400 


## Changes proposed in this Pull Request:
- remove Link beta headers on checkout.







## Testing instructions


- Navigate to Settings -> Payment Methods and enable Link payment method.
- create an order using Link payment method
- the order should be completed without errors


---


**Post merge**
